Maquetado/scss/titles.scss

```diff
@@ -0,0 +1,20 @@
+$urlImagen: "./assets/triangle.svg";
+
+.title {
+  position: relative;
+
+  .img {
+    background-image: url($urlImagen);
+    background-repeat: no-repeat;
+    margin: 10px 0 0 10px;
+    padding: 60px;
+    position: absolute;
+  }
+
+  .text {
```

Comment:
the h2 should have the class `title` and this properties should be placed there.